roles/hello-world-apb/vars/deprovision.yml

```diff
@@ -0,0 +1,2 @@
+---
+state: absent
```

Comment:
Seems a little overkill to have a whole vars directory for these two 1-line files.

Reply:
I agree. I really just wanted to pull vars out of the playbook but I don't think this improves on that.